### PR TITLE
docs: add changelog 4.16.10

### PR DESCRIPTION
**Pull request by vagusX**
中文

- 🐞 修复 Badge 设置自定义 `color` 不显示数字的问题。[#31617](https://github.com/ant-design/ant-design/pull/31617) [@eokoneyo](https://github.com/eokoneyo)
- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ ant-design/ant-design/pull/31589)
- 🐞 修复 Select 组件没有忽略 `getRawInputElement` 属性导致的类型报错问题。[#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
- 🐞 修复 Pagination 的 `totalBoundaryShowSizeChanger` 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
- 🐞 修复 Skeleton.Avatar `className` 重复应用的问题。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
- 💄 修复 Form 内 `<Input bordered={false} />` 错误状态下有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
- 🌐 国际化
  - 🇹🇷 为 Image 组件中 `预览` 文案增加土耳其语翻译。[#31593](https://github.com/ant-design/ant-design/pull/31593) [@mburakkalkan](https://github.com/mburakkalkan)
  - 🇰🇷 修复韩语中的错别字。[#31575](https://github.com/ant-design/ant-design/pull/31575) [@chatoo2412](https://github.com/chatoo2412)

----------

英文

- 🐞 Fix Badge not showing count when customize `color` prop. [#31617](https://github.com/ant-design/ant-design/pull/31617) [@eokoneyo](https://github.com/eokoneyo)
- 🐞 Fix circle Progress `success.strokeColor` cannot take effect. [#31589](https://github.com/ant-design/ant-design/pull/31589)
- 🐞 Fix the type error due to the Select component didn't ignore the `getRawInputElement` prop. [#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
- 🐞 Fix Pagination's `totalBoundaryShowSizeChanger` property type error. [#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
- 🐞 Fix `className` applied to both Skeleton.Avatar wrapper and element itself. [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
- 💄 Fix Form `<Input bordered={false} />` outline style. [#31562](https://github.com/ant-design/ant-design/pull/31562)
- 🌐 Internationalization
  - 🇹🇷 Added Turkish translation for Image `Preview` text. [#31593](https://github.com/ant-design/ant-design/pull/31593) [@mburakkalkan](https://github.com/mburakkalkan)
  - 🇰🇷 Fix a typo in the Korean language. [#31575](https://github.com/ant-design/ant-design/pull/31575) [@chatoo2412](https://github.com/chatoo2412)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ timeline: true
...
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 Fix circle Progress `success.strokeColor` cannot take effect. [#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 Fix Pagination's totalBoundaryShowSizeChanger property type error. [#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 Fix `className` applied to both SkeletonAvatar wrapper and element itself. [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
+- 💄 Fix Form `&lt;Input bordered={false} /&gt;` outline style. [#31562](https://github.com/ant-design/ant-design/pull/31562)
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:25]:
```suggestion
- 💄 Fix Form `<Input bordered={false} />` outline style. [#31562](https://github.com/ant-design/ant-design/pull/31562)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ timeline: true
...
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 修复 Pagination 的 totalBoundaryShowSizeChanger 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 修复应用于 SkeletonAvatar wrapper 和元素本身的 `className`。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
+- 💄 修复 Form 内 `&lt;Input bordered={false} /&gt;` 错误样式有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:25]:
```suggestion
- 💄 修复 Form 内 `<Input bordered={false} />` 错误样式有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
```

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/31617
- [ ] https://github.com/ant-design/ant-design/pull/31566

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,18 @@ timeline: true
 
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 修复 Pagination 的 totalBoundaryShowSizeChanger 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 修复应用于 SkeletonAvatar wrapper 和元素本身的 `className`。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🐞 修复 Skeleton.Avatar `className` 重复应用的问题。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
```

**Comment by vagusX** [comment]
> * [ ]  [fix: Badge count style when customize color  #31617](https://github.com/ant-design/ant-design/pull/31617)
> * [ ]  [fix(select): omit getRawInputElement #31566](https://github.com/ant-design/ant-design/pull/31566)

这两个也一起发嘛？

**Comment by afc163** [comment]
修复 formily 中使用 Select 没有忽略 `getRawInputElement` 属性导致的定义报错问题。

**Comment by vagusX** [comment]
> 修复 formily 中使用 Select 没有忽略 `getRawInputElement` 属性导致的定义报错问题

- 修复 Select 组件没有忽略 getRawInputElement prop 导致的类型报错问题。
- Fix the type error due to the Select component didn't ignore the `getRawInputElement` prop.

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,18 @@ timeline: true
 
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 修复 Pagination 的 totalBoundaryShowSizeChanger 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🐞 修复 Pagination 的 `totalBoundaryShowSizeChanger` 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
```

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 Fix Badge not showing count when customize color prop. [#31617](https://github.com/ant-design/ant-design/pull/31617)
+- 🐞 Fix circle Progress `success.strokeColor` cannot take effect. [#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 Fix Pagination's totalBoundaryShowSizeChanger property type error. [#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 Fix `className` applied to both SkeletonAvatar wrapper and element itself. [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
+- 💄 Fix Form `<Input bordered={false} />` outline style. [#31562](https://github.com/ant-design/ant-design/pull/31562)
+- 🌐 Internationalization
+  - Added Turkish translation for Image 'Preview' text. [#31593](https://github.com/ant-design/ant-design/pull/31593) [@mburakkalkan](https://github.com/mburakkalkan)
+  - Fix a typo in the Korean language. [#31575](https://github.com/ant-design/ant-design/pull/31575) [@chatoo2412](https://github.com/chatoo2412)
```
Comment (afc163) [inline CHANGELOG.en-US.md:30]: 增加 🇰🇷 🇹🇷 的 emoji。

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+`2021-08-02`
+
+- 🐞 修复 Select 组件没有忽略 `getRawInputElement` prop 导致的类型报错问题。[#31566](https://github.com/ant-design/ant-design/pull/31566)
+- 🐞 修复 Badge 设置自定义 `color` 不显示数字的问题。[#31617](https://github.com/ant-design/ant-design/pull/31617)
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 修复 Pagination 的 totalBoundaryShowSizeChanger 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 修复 Skeleton.Avatar `className` 重复应用的问题。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
+- 💄 修复 Form 内 `<Input bordered={false} />` 错误样式有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
+- 🌐 国际化
+  - 为 Image 组件中 "预览" 文案增加土耳其语翻译 [#31593](https://github.com/ant-design/ant-design/pull/31593) [@mburakkalkan](https://github.com/mburakkalkan)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]:
```suggestion
  - 为 Image 组件中 `预览` 文案增加土耳其语翻译。[#31593](https://github.com/ant-design/ant-design/pull/31593) [@mburakkalkan](https://github.com/mburakkalkan)
```

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 Fix Badge not showing count when customize color prop. [#31617](https://github.com/ant-design/ant-design/pull/31617)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:22]: ``` `color` ```

File: CHANGELOG.en-US.md [hunk 8]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 Fix circle Progress `success.strokeColor` cannot take effect. [#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 Fix the type error due to the Select component didn't ignore the `getRawInputElement` prop. [#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
+- 🐞 Fix Pagination's totalBoundaryShowSizeChanger property type error. [#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:25]: ``` `totalBoundaryShowSizeChanger` ```

File: CHANGELOG.en-US.md [hunk 9]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 Fix Badge not showing count when customize `color` prop. [#31617](https://github.com/ant-design/ant-design/pull/31617)
+- 🐞 Fix circle Progress `success.strokeColor` cannot take effect. [#31589](https://github.com/ant-design/ant-design/pull/31589)
+- 🐞 Fix the type error due to the Select component didn't ignore the `getRawInputElement` prop. [#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
+- 🐞 Fix Pagination's `totalBoundaryShowSizeChanger` property type error. [#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 Fix `className` applied to both SkeletonAvatar wrapper and element itself. [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
```
Comment (afc163) [inline CHANGELOG.en-US.md:26]:
```suggestion
- 🐞 Fix `className` applied to both Skeleton.Avatar wrapper and element itself. [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
```

File: CHANGELOG.zh-CN.md [hunk 10]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 修复 Badge 设置自定义 `color` 不显示数字的问题。[#31617](https://github.com/ant-design/ant-design/pull/31617)
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ ant-design/ant-design/pull/31589)
+- 🐞 修复 Select 组件没有忽略 `getRawInputElement` prop 导致的类型报错问题。[#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🐞 修复 Select 组件没有忽略 `getRawInputElement` 属性导致的类型报错问题。[#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
```

File: CHANGELOG.zh-CN.md [hunk 11]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 4.16.10
+
+`2021-08-02`
+
+- 🐞 修复 Badge 设置自定义 `color` 不显示数字的问题。[#31617](https://github.com/ant-design/ant-design/pull/31617)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:22]:
```suggestion
- 🐞 修复 Badge 设置自定义 `color` 不显示数字的问题。[#31617](https://github.com/ant-design/ant-design/pull/31617) [@eokoneyo](https://github.com/eokoneyo)
```

还是带上原修改者。

File: CHANGELOG.zh-CN.md [hunk 12]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 修复 Progress 环形进度条 `success.strokeColor` 不生效的问题。[#31589](https://github.com/ ant-design/ant-design/pull/31589)
+- 🐞 修复 Select 组件没有忽略 `getRawInputElement` prop 导致的类型报错问题。[#31566](https://github.com/ant-design/ant-design/pull/31566) [@aoilti](https://github.com/aoilti)
+- 🐞 修复 Pagination 的 `totalBoundaryShowSizeChanger` 属性类型错误。[#31549](https://github.com/ant-design/ant-design/pull/31549) [@Monty-Ma](https://github.com/Monty-Ma)
+- 🐞 修复 Skeleton.Avatar `className` 重复应用的问题。 [#31536](https://github.com/ant-design/ant-design/pull/31536) [@Greatshock](https://github.com/Greatshock)
+- 💄 修复 Form 内 `<Input bordered={false} />` 错误样式有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:27]:
```suggestion
- 💄 修复 Form 内 `<Input bordered={false} />` 错误状态下有异常外框样式的问题。[#31562](https://github.com/ant-design/ant-design/pull/31562)
```

**Review by afc163 — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

**Pull request merged**